### ISSUE TITLE
[Feature Request] see commit msg for updates

### ISSUE THREAD

**Issue by eleanorkonik** [ganesshkumar/obsidian-plugins-stats-ui#3]
First, this is incredible, thank you for building this. 

**Is your feature request related to a problem? Please describe. **

Seeing recent updates for plugins has always relied on developers to self-report, it's always been the hardest part of things to keep up with. I _love_ that you've created a way for me to see the most recent updates — but I figured I'd ask if it's possible to include the update information or commit message or something in the feed so that it's possible to get a hint of what the update was about? 

This would make it a lot easier for me to amplify updates from developers who don't use Discord. 

**Describe the solution you'd like**

For updates it would be nice if the github repo description got replaced with the commit message or release notes for a plugin. 

**Describe alternatives you've considered**

- continuing to rely on developers to self-report
- downloading every single plugin into a test vault and then using the changelog plugin to track things has occurred to me, but it seemed really unwieldy as a method so I haven't done it. 

**Additional context**

I run the [Obsidian Roundup](https://obsidianroundup.org/). Thank you!!

**Comment by ganesshkumar** [comment]
@eleanorkonik,  thanks for bringing this up! 

I tried adding the release notes from the latest release into the UI but 
- different plugins had varying length of release.   
- some of them were very huge that it took almost half of the screen when restricting the width to that of the card.   
-different plugins have different text formats in the release notes - some of the plugins have markdown content and some have plain text. 


Current Behavior: 
- No description
- Clicking on cards directly redirects us to the GitHub repo.

Proposed Behavior: 
- [Option 1] Cards have a 'view changelog' link
    - [Option 2] Alternatively, we can truncate the text to first x characters and give a 'read more' link.
- Clicking on the 'view changelog' link will take us to a plugin page (has to be built; where all the details of the plugin - latest release notes, total downloads etc. can be seen) and 
- Give a link to the GitHub repo from the plugin page. 

Let me know your thoughts.

**Comment by eleanorkonik** [comment]
I prefer option 2 (it might encourage devs to give good one-liner intros for their release notes ;) 

and this would be _awesome_. 

I can already see so many ways to use this with the roundup — I could highlight the biggest / most important 10 or so updates and then link to your site for people to see the rest of them, for example.

**Comment by ganesshkumar** [comment]
I tried to truncate the content but it seems to be little tricky (few changelogs begins with image. It fails when truncating the markdown syntax of the image. Alternatively, it will take some time to render it to html and extract the elements that can be displayed. 

I will continue to explore this issue. Meanwhile, as a quick fix does the following idea help? 
In the `updates` tab, where plugins are sorted by latest release time, I have added a `changelog` link which is collapsible content. We can click to text, to expand and see the entire changelog along with pics below. We can click it again to collapse the content. 

![animation](https://user-images.githubusercontent.com/2135089/154822083-4a01f616-08cc-4b78-ae5d-700e166f926b.gif)

Let me know if this works until I figure out how to effiective clip the text content alone and put it in the card.
